logging/README.md

```diff
@@ -257,6 +257,48 @@ The script creates the namespace into which the components are deployed. By defa
 Updates in place are supported. To update, re-run the
 `deploy_logging_open.sh` script to install the latest versions of all components, indexes, and dashboards.
 
+## Important Information about Kibana in the New Release
+
+### Overview
+
+**Notes:**
+
+* As of release 1.1.0, this project now uses Open Distro for Elasticsearch version 1.13.2, which includes updated versions of Elasticsearch and Kibana.
+* In this release, the multi-tenancy capabilities of Kibana have been enabled. This change requires that users select a Kibana tenant space after logging in. For the new procedure, see [Log Into Kibana](#login_kibana).
+* Kibana no longer provides a persistent navigation bar by default. Instead, a menu button ( ![menu button](../img/kibana_menu.png) ) that opens the navigation menu is now provided. You can choose to make the navigation menu persist on the page by clicking __Dock navigation__.
+
+### <a name="login_kibana"></a>Log In to Kibana
+
+Use the following steps to log in to Kibana:
+
+1. In the "Login to Kibana" window, enter your Kibana credentials and click __Log In__. The "Select your tenant" window appears.
+2. In the __Choose from custom__ list, confirm that your tenant is selected by default. If not, select your tenant from the list.
```

Comment:
I wonder if we should use special formatting for the window titles...to make them stand-out more.  Maybe bold + italic (to differentiate them from the menu text which we have used bold for).

Reply:
Introducing more typeface formatting is poor usability. Again, the formatting follows SAS Style Guidelines.